client/settings/express-checkout-settings/general-payment-request-button-settings.js

```diff
@@ -136,34 +140,36 @@ const GeneralPaymentRequestButtonSettings = ( { type } ) => {
 	}, [] );
 
 	const otherButtons =
-		'woopay' === type ? 'Apple Pay / Google Pay' : 'WooPay';
+		'woopay' === type ? 'Apple Pay / Google Pay buttons' : 'WooPay button';
```

Comment:
Should we make these translatable?

```suggestion
		'woopay' === type ? __( 'Apple Pay / Google Pay buttons', 'woocommerce-payments' ) : __( 'WooPay button', 'woocommerce-payments' );
```